### PR TITLE
Vagrantfile default to no monitoring

### PR DESCRIPTION
**Pull request by martinb3**

### CODE REVIEW

**Comment by hhoover** [comment]
should the Vagrantfile exist at all? We're testing with TK...

**Comment by martinb3** [comment]
@hhoover Probably not, but I made that change before we discussed it.

**Comment by hhoover** [comment]
@prometheanfire merged before discussion was over... nice. :-1:

File: recipes/monitors.rb [hunk 1]
```diff
@@ -76,6 +76,7 @@
     variables(
       cookbook_name: cookbook_name
     )
+    only_if node['platformstack']['cloud_monitoring']['enabled'] == true
```
Comment (gondoi) [inline recipes/monitors.rb:79]: I keep seeing this sytax. Is it actually working? I've always understood that any ruby checks like this needed to be passed as a ruby block in `{}`.

Comment (hhoover) [inline recipes/monitors.rb:79]: Yep. see #70

Comment (martinb3) [inline recipes/monitors.rb:79]: The final commit "Always run platformstack::monitors, just only_if" was not meant to be in the PR. :/ PS -- it wasn't working before this either. No one had tested turning this attribute off. It caused a failure to converge.

**Pull request merged**